tests/test_telemetry_plugin.py

```
@@ -799,7 +816,155 @@ async def run_test():
+                    "!r", event.event_id, "❌"
+                )
+
+            asyncio.run(run_test())
+
+    @patch("mmrelay.matrix_utils.connect_matrix")
+    @patch("mmrelay.matrix_utils.upload_image")
+    @patch("mmrelay.matrix_utils.send_room_image")
+    @patch("mmrelay.plugins.telemetry_plugin.plt.xticks")
+    @patch("mmrelay.plugins.telemetry_plugin.plt.subplots")
+    def test_handle_room_message_calculate_averages_non_dict_record(
+        self, mock_subplots, mock_xticks, mock_send_image, mock_upload, mock_connect
```

Ruff (0.15.10)

[warning] Unused method argument: `mock_xticks`

(ARG002)